packages/nuxt/package.json

```diff
@@ -26,16 +26,16 @@
   "sideEffects": false,
   "exports": {
     ".": {
-      "require": "./dist/index.js",
-      "import": "./dist/index.mjs"
+      "import": "./dist/module.mjs",
```

Comment:
shouldn't this be renamed to `./dist/index.cjs`? that way siroc will generate the file